src/Umbraco.Web/PublishedCache/NuCache/PublishedSnapshotService.cs

```diff
@@ -217,7 +223,12 @@ private void LoadCachesOnStartup()
         {
             var okContent = false;
             var okMedia = false;
-
+            if (_syncBootStateAccessor.GetSyncBootState() == SyncBootState.ColdBoot)
+            {
+                _logger.Warn<PublishedSnapshotService>("Sync Service is in a Cold Boot state. Skip LoadCachesOnStartup as the Sync Service will trigger a full reload");
```

Comment:
This should just be info, else we have 2x warnings for every cold boot when cold boots are actually 'normal'